student.rb

```diff
@@ -9,4 +11,8 @@ def initialize(id, age, name, classroom)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def belongs_to?
+    @classroom = self
+  end
```

Comment:
- [ ] Please follow the steps below :point_down:  to create the correct method 

```suggestion
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

- **def classroom=(classroom):**

  - This defines a method named classroom= which takes one argument, classroom.
- **@classroom = classroom:**

  - This assigns the value of the classroom argument to the instance variable @classroom.
- **classroom.students.push(self) unless classroom.students.include?(self):**

  - **This line has two parts:**
    - **classroom.students.push(self):** It adds the current instance (self) to the list of students in the provided classroom.
    - **unless classroom.students.include?(self):** This part checks whether the current instance (self) is already present in the list of students for the provided classroom. If it's not present, the push operation is performed to add it to the list.

In summary, this method is used to set the classroom for a student instance and also ensures that the student is added to the list of students for that classroom if they are not already present. It prevents adding the same student to the classroom's student list multiple times.